src/Sylius/Bundle/AdminBundle/Action/RemoveAvatarAction.php

```diff
@@ -40,12 +36,10 @@ final class RemoveAvatarAction
 
     public function __construct(
         AvatarImageRepositoryInterface $avatarRepository,
-        EngineInterface $templatingEngine,
```

Comment:
Isn't it a bc break? :(

Reply:
Good catch 👍 Kind of, clarifying it here: https://github.com/Sylius/Sylius/pull/10844